src/pages/clients/common/hooks/useClientColumns.tsx

```diff
@@ -0,0 +1,346 @@
+    },
+    {
+      column: 'created_at',
+      id: 'created_at',
+      label: t('created_at'),
+      format: (value) => date(value, dateFormat),
+    },
+    {
+      column: 'credit_balance',
+      id: 'credit_balance',
+      label: t('credit_balance'),
+      format: (value, client) =>
+        formatMoney(value, client?.country_id, client?.settings.currency_id),
+    },
+    {
+      column: 'currency',
+      id: 'id',
+      label: t('currency'),
+      format: (value, client) =>
+        client.settings?.currency_id &&
+        resolveCurrency(client.settings.currency_id)?.code,
+    },
+    {
+      column: 'custom1',
+      id: 'custom_value1',
+      label:
+        (company?.custom_fields.client1 &&
+          customField(company?.custom_fields.client1).label()) ||
+        t('first_custom'),
+    },
+    {
+      column: 'custom2',
+      id: 'custom_value2',
+      label:
+        (company?.custom_fields.client2 &&
+          customField(company?.custom_fields.client2).label()) ||
+        t('second_custom'),
+    },
+    {
+      column: 'custom3',
+      id: 'custom_value3',
+      label:
+        (company?.custom_fields.client3 &&
+          customField(company?.custom_fields.client3).label()) ||
+        t('third_custom'),
+    },
+    {
+      column: 'custom4',
+      id: 'custom_value4',
+      label:
+        (company?.custom_fields.client4 &&
+          customField(company?.custom_fields.client4).label()) ||
+        t('forth_custom'),
+    },
+    {
+      column: 'documents',
+      id: 'documents',
+      label: t('documents'),
+      format: (value, client) => client.documents.length,
+    },
+    {
+      column: 'entity_state',
+      id: 'id',
+      label: t('entity_state'),
+      format: (value, client) => <EntityStatus entity={client} />,
+    },
+    {
+      column: 'id_number',
+      id: 'id_number',
+      label: t('id_number'),
+    },
+    {
+      column: 'is_deleted',
+      id: 'is_deleted',
+      label: t('is_deleted'),
+      format: (value, client) => (client.is_deleted ? t('Yes') : t('No')),
+    },
+    {
+      column: 'language',
+      id: 'id',
+      label: t('language'),
+      format: () => '', // @Todo: Need to resolve
+    },
+    {
+      column: 'phone',
+      id: 'phone',
+      label: t('phone'),
+    },
+    {
+      column: 'private_notes',
+      id: 'private_notes',
+      label: t('private_notes'),
+      format: (value) => <span className="truncate">{value}</span>,
+    },
+    {
+      column: 'public_notes',
+      id: 'public_notes',
+      label: t('public_notes'),
+      format: (value) => <span className="truncate">{value}</span>,
+    },
+    {
+      column: 'state',
+      id: 'state',
+      label: t('state'),
+    },
+    {
+      column: 'address1',
+      id: 'address1',
+      label: t('address1'),
+    },
+    {
+      column: 'task_rate',
+      id: 'id',
+      label: t('task_rate'),
+      format: () => '', // @Todo: Need to resolve
```

Comment:
@turbo124 what's the value of task rate? I can't see the column.

Reply:
client.settings.default_task_rate if it exists, or company.settings.default_task_rate